dtfs-central-api/src/types/tfm/team.ts

```diff
@@ -0,0 +1,15 @@
+export type TeamId =
+  | 'UNDERWRITER_SUPPORT'
```

Comment:
Good catch! Will update. We can't currently generate the type from `trade-finance-manager-api\src\constants\teams.js` though as it's part of a different package, and there's no real benefit of redeclaring those constants in `dtfs-central-package` because it never does any logic on them.

At some point soon we'll be introducing a common package that can share code across the codebase, and this type would be exactly the sort of thing we would have there so we can ensure consistency across the whole project.